src/browser/rollbar.js

```diff
@@ -0,0 +1,269 @@
+var Client = require('../rollbar');
+var _ = require('../utility');
+
+var transforms = require('./transforms');
+var predicates = require('./predicates');
+var errorParser = require('./errorParser');
+
+function Rollbar(options, client) {
+  this.options = _.extend(true, defaultOptions, options);
+  var context = 'browser';
+  this.client = client || new Client(context, this.options);
+  this.init(this.client);
+}
+
+Rollbar.prototype.global = function(options) {
+  this.client.global(options);
```

Comment:
Ah, I see that this is configuring the RateLimiter singleton. Do you think we should rename this method or move the configuration for the global rate limiter to no longer be accessible via each Rollbar instance?